vue-frontend/src/components/portfolio/BrandingSection.vue

```diff
@@ -158,22 +158,6 @@
                   :jsonData="data.animation"
                   class="tw-mt-14 lg:tw-mt-20"
                 />
-                <video
```

Comment:
togness video is not aligned with the image. Digita will provide you video, add it instead of image and video